asl_xdsl/dialects/asl.py

```diff
@@ -151,7 +151,33 @@ def parse_parameter(cls, parser: AttrParser) -> bool:
 
     def print_parameter(self, printer: Printer) -> None:
         """Print the attribute parameter."""
-        printer.print("true" if self.data else "false")
+        printer.print("<true>" if self.data else "<false>")
+
+
+@irdl_attr_definition
+class StringType(ParametrizedAttribute, TypeAttribute):
+    """A string type."""
+
+    name = "asl.string"
+
+
+@irdl_attr_definition
+class StringAttr(Data[str]):
```

Comment:
I would recommend using the StringAttr in the builtin dialect

Reply:
That's simpler - done